sorc/chgres_cube.fd/docs/user_guide.md

```diff
@@ -1,11 +1,383 @@
-
 # chgres_cube
 
-Creates cold start initial conditions for FV3 model runs. This is part
-of the [NCEPLIBS UFS_UTILS](https://github.com/NOAA-EMC/UFS_UTILS) project.
+# Introduction
+
+The chgres_cube program creates initial condition files to “coldstart”
+the forecast model. The initial conditions are created from either
+Global Forecast System (GFS) gridded binary version 2 (GRIB2), NOAA
+Environmental Modeling System Input/Output (NEMSIO) data, or Network
+Common Data Form (NetCDF) data.
+
+This is part of the [NCEPLIBS
+UFS_UTILS](https://github.com/NOAA-EMC/UFS_UTILS) project.
+
+This user guide is part of the <a href="../index.html">UFS_UTILS
+documentation</a>.
+
+## Where to find GFS GRIB2, NEMSIO and NetCDF data
```

Comment:
Could you just link to the 'readthedocs' here?  If the 'rst' file changes, then the same change would have to be made here.

Reply:
My intention is to go the other way around, so that the doxygen generates the readthedocs output automatically.